resources/walkthrough/flink/statements.md

```diff
@@ -1,5 +1,3 @@
-> **Note:** Flink support is currently in **preview**, enable it in the extension settings before using these features.
```

Comment:
I believe we are still keeping the UDFs/Artifacts under preview, so we will need a nod to that here